src/status_im/ui/screens/wallet/events.cljs

```diff
@@ -51,14 +60,16 @@
 
 (handlers/register-handler-fx
   :refresh-wallet
-  (fn [{{:keys [web3 accounts/current-account-id] :as db} :db} [_ a]]
+  (fn [{{:keys [web3 accounts/current-account-id network] :as db} :db} [_ a]]
     {:get-balance {:web3          web3
                    :account-id    current-account-id
                    :success-event :update-balance
                    :error-event   :update-balance-fail}
+     :get-transactions {:account-id current-account-id
```

Comment:
Transactions should be fetched separately for 2 reasons:

* we don't want to load too much upfront if this is not needed
* we want to refresh individually transactions (transactions list will support pull-to-refresh)